pyproject.toml

```diff
@@ -23,25 +22,29 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
+python = ">=3.9.2,<3.13"
+pylint = "^3.3.6"
```

Comment:
I don't know that we want to push to this new of a version, just doing it for testing.